### ISSUE TITLE
docs.rs build is failing for 0.12.0

### ISSUE THREAD

**Issue by metasim** [georust/gdal#268]
<img width="500" alt="image" src="https://user-images.githubusercontent.com/131013/163739362-b174651f-b3da-40b3-a504-592b52bcc3fa.png">

Looks like some sort of linker error:

```
/opt/rustwide/workdir/build.rs:13: undefined reference to `GDALVersionInfo'
```

Build log:

https://docs.rs/crate/gdal/0.12.0/builds/494924

**Comment by metasim** [comment]
Perhaps `DOCS_RS` is no longer defined? 🤷 

https://github.com/georust/gdal/blob/492e0c8e3f14c26e0e3442e42a04c97c14ff627a/gdal-sys/build.rs#L63-L66

**Comment by rmanoka** [comment]
related to #240 ?

**Comment by metasim** [comment]
Closed by #281
